src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs

```diff
@@ -55,8 +55,6 @@ public class WpfHost : WpfControl, WinUI.ISkiaHost, IWpfHost
 		private readonly Func<UnoApplication> _appBuilder;
 		private CompositeDisposable _registrations = new();
 
-		private bool _appStarted = false;
-		
```

Comment:
@MartinZikmund This was never set to true.

I kept the existing behavior and just removed the always-false field.
If this is a bug and there should be `_appStarted = true` in `StartApp`, let me know.

Reply:
That's fine, let's keep the field removed.